client/src/components/profile/components/TimeLine.tsx

```diff
@@ -163,7 +163,10 @@ class TimelineInner extends Component<TimelineInnerProps, TimeLineInnerState> {
     }
   }
 
-  renderCompletion(completed: SortedTimeline): JSX.Element {
+  renderCompletion(
+    completed: SortedTimeline,
+    t: TFunction<'translation'>
```

Comment:
Need to pass the `t` from `props` to avoid the app crashing from a hook.

Reply:
If you feel a sudden rush of enthusiasm, it would be good to convert this to a functional component at some point.  Maybe a separate PR to avoid scope creep.